From f1750e9a222b440d054e1be1dfef9ad3ae28be24 Mon Sep 17 00:00:00 2001
From: Jerry Cao <jerry.cao@amlogic.com>
Date: Fri, 11 Apr 2014 18:03:40 -0700
Subject: [PATCH 3931/5965] Print out reserved memory address as fixed length
 hex number.

Make the hex number printed out fixed 8 digit long, with 0 padding
in front to ease reading of the output.
---
 drivers/of/fdt.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/of/fdt.c b/drivers/of/fdt.c
index b75f328ed729..e2e573050c09 100755
--- a/drivers/of/fdt.c
+++ b/drivers/of/fdt.c
@@ -886,7 +886,7 @@ int __init early_init_dt_scan_memory(unsigned long node, const char *uname,
 #endif
 
 	pr_info("Total memory is %4d MiB\n",((unsigned int)total >> 20));
-	pr_info("Reserved low memory from 0x%llx to 0x%llx, size: %3d MiB \n",
+	pr_info("Reserved low memory from 0x%08llx to 0x%08llx, size: %3d MiB \n",
 		aml_reserved_start,aml_reserved_end,
 		((unsigned long)(aml_reserved_end - aml_reserved_start + 1)) >> 20);
 
-- 
2.19.0

